docs/ens-spec.md

```diff
@@ -0,0 +1,18 @@
+# Resource Identifier (RI) Scheme name: ens
+
+# Scheme syntax
+
+```ens://v?r=[region]&c=[verificaton code]```
+
+# Scheme semantics
+
+Send covid-19 exposure notifications verifications code into a compatible mobile application or operating system for verificafication of a diagnosis to share within the exposure notifications system.
```

Comment:
Some typos here: Send covid-19 exposure notifications **verification codes** into a compatible mobile application or operating system for **verification** of a diagnosis to share within the exposure notifications system.